modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_search_iterator.scss

```diff
@@ -138,6 +138,10 @@
 	.lfr-asset-item {
 		width: 50%;
 
+		@include media-breakpoint-up(xs) {
```

Comment:
We can get rid of this whole block and just change `width: 50%;` => `width: 100%;` The media query generated by `@include media-breakpoint-up(xs)` is `@media (min-width: 0)`.